.travis.yml

```diff
@@ -0,0 +1,15 @@
+sudo: false
```

Comment:
How about using the Stack-recommended Travis config to get a much more inclusive matrix?

* https://docs.haskellstack.org/en/stable/travis_ci/
* https://raw.githubusercontent.com/commercialhaskell/stack/master/doc/travis-complex.yml